docs/source/development/pack_testing.rst

```diff
@@ -0,0 +1,98 @@
+:orphan:
+
+Pack Testing
+============
+
+This section includes information on pack testing - where to put the tests,
+how to write the tests, mock classes which can be used to make testing
+easier, etc.
+
+Test File Locations and Names
+-----------------------------
+
+All the test files should go into ``<pack name>/tests/`` directory. If tests
+include any fixtures, they should be put in the ``<pack name>/tests/fixtures/``
+directory.
```

Comment:
Would it make sense to also add a `<pack name>/mocks/` directory as a convention?  These could be used to mock sessions to remote services for example.